bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -284,6 +300,14 @@ public void batch(List<? extends Row> actions, @Nullable Object[] results)
         hosts.add(options.getDataHost());
       }
     }
+    if (removeSucceededActions) {
+      for (int i = results.length - 1; i >= 0; i--) {
+        // if result is not null, it succeeded
+        if (results[i] instanceof Result) {
```

Comment:
I'm a bit confused by this check. Isnt the result type for a delete a Void? and Result is for Get?

Reply:
Hmm, I thought it would be an empty result? https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java#L107

I could also check if it's not a throwable

Reply:
I think the later would be better